### PR TITLE
Stopover fix 

### PR DESCRIPTION
**Pull request by TheGiraffe3**
Fixes some problems with scientist missions.

### CODE REVIEW

File: data/successor jobs.txt [hunk 1]
```diff
@@ -736,4 +736,4 @@ mission "High Houses: Criminal Capture 1"
 		dialog phrase "generic bounty hunting on visit"
 	on complete
 		payment 140000
-		dialog phrase "generic bounty hunting payment dialog"
\ No newline at end of file
+		dialog phrase "generic bounty hunting payment dialog"
```
Comment (TheGiraffe3) [inline data/successor jobs.txt:760]: No idea why this shows as changed.

**Comment by TheGiraffe3** [comment]
Ready.

**Comment by TheGiraffe3** [comment]
@Daeridanii1 what do you think about this?

File: data/successor jobs.txt [hunk 2]
```diff
@@ -87,7 +87,7 @@ phrase "predecessor researchers out"
 		`scientists `
 		`scholars `
 	word
-		`to <planet> where they `
+		`to <stopovers> where they `
```
Comment (Daeridanii1) [inline data/successor jobs.txt]:
```suggestion
		`to <planet stopovers> where they `
```

File: data/successor jobs.txt [hunk 3]
```diff
@@ -119,7 +119,7 @@ phrase "successor researchers out"
 		`scientists `
 		`experimenters `
 	word
-		`to <planet> where they `
+		`to <stopovers> where they `
```
Comment (Daeridanii1) [inline data/successor jobs.txt]:
```suggestion
		`to <planet stopovers> where they `
```

**Review by Daeridanii1 — COMMENTED** [review]
\<stopovers\> has the "planet in the system" format, while \<planet stopovers\> just has the "planet" format, which I think is more appropriate here.

**Pull request merged**